src/reconcile/rec_write.c

```diff
@@ -6591,3 +6594,29 @@ __rec_dictionary_lookup(
 	*dpp = next;
 	return (0);
 }
+
+/*
+ * __wt_verbose_lookaside_write --
+ * Create a verbose message to display with details about the transaction state
+ * when performing a lookaside table write.
+ */
+void
+__wt_verbose_lookaside_write(WT_SESSION_IMPL *session, int64_t inserted_cnt)
+{
+	WT_SESSION* wt_session;
+	uint32_t pct_dirty, pct_full;
+	uint64_t pinned_range;
+
+	wt_session = &session->iface;
+
+	__wt_eviction_clean_needed(session, &pct_full);
+	__wt_eviction_dirty_needed(session, &pct_dirty);
+	wt_session->transaction_pinned_range(wt_session, &pinned_range);
+
+	__wt_verbose(session, WT_VERB_LOOKASIDE,
+	    "Lookaside written to. Number of TXN ID's Pinned: %" PRIu64
+	    ". Cache is %" PRIu32 "%% dirty. "
+	    "Cache is %" PRIu32 "%% full. "
+	    "Page entries requiring LAS support: %" PRId64,
+	    pinned_range, pct_dirty, pct_full, inserted_cnt);
+}
```

Comment:
Can we make the message read naturally, instead of being so choppy? Off the top of my head, something like:
```
__wt_verbose(session, WT_VERB_LOOKASIDE,
    "Page reconciliation required %" PRId64 " lookaside entries: %" 
    PRIu64 " pinned TXN ID's, cache is %" PRIu32 "/%" PRIu32 "%% dirty/clean",
    inserted_cnt, pinned_range, pct_dirty, pct_full);
```
Gets rid of "LAS", too. :)